app/views/login.handlebars

```diff
@@ -0,0 +1,18 @@
+<html>
+{{#if message}}
+<div class="error">
+  <span style="color: red">{{message}}</span>
+</div>
+{{/if}}
+<form action="/login" method="post">
+  <div>
+      <label>Username:</label>
```

Comment:
Minor nit, but sometimes the indentation seems to be two spaces and other times (like here) it seems like it's four.